src/types/Authorization.ts

```diff
@@ -84,6 +84,8 @@ export enum Entity {
   LOGGINGS = 'Loggings',
   LOGGING = 'Logging',
   PRICING = 'Pricing',
+  PRICING_DEFINITION = 'PricingDefinition',
+  PRICING_DEFINITIONS = 'PricingDefinitions',
```

Comment:
Maybe we should have used only singular entities to simplify :-)
But that's another story!